pxr/base/arch/testenv/testFileSystem.cpp

```diff
@@ -92,6 +114,12 @@ int main()
     fclose(firstFile);
     ARCH_AXIOM(ArchGetFileLength(firstName.c_str()) == strlen(testContent));
 
+    // Open a file, check that the file path from FILE* handle is matched.
+    ARCH_AXIOM((firstFile = ArchOpenFile(firstName.c_str(), "rb")) != NULL);
+    std::string filePath = ArchGetFileName(firstFile);
+    ARCH_AXIOM(_DosDevicePathFilter(filePath) == _DosDevicePathFilter(firstName));
```

Comment:
Would comparing `ArchAbsPath` be sufficient instead of `_DosDevicePathFilter`?

Reply:
It doesn't work to me.